ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -291,6 +291,11 @@ def odf_overview_ui(
                     self.validation_loc["storage-system-status-card-hyperlink"],
                     enable_screenshot=True,
                 )
+            # verify that only one BlockPools tab is present. BZ #2096513
+            blockpools_tabs = self.get_elements(self.validation_loc["blockpools"])
+            if len(blockpools_tabs) > 1:
+                logger.critical("Multiple BlockPools tabs were found. BZ #2096513")
```

Comment:
Shouldn't we fail the test here?

Reply:
having warnings.warn next line the test will proceed until the end and fail in the end with proper output

Reply:
I don't think it's necessary to proceed if the test is failing at this point